src/ImageTransformations.jl

```diff
@@ -1,93 +1,28 @@
 __precompile__()
 module ImageTransformations
 
-using CoordinateTransformations, Interpolations, OffsetArrays, StaticArrays, FixedPointNumbers, Colors, ColorVectorSpace, ImageCore
-
-import Base: start, next, done, eltype, iteratorsize
+using ImageCore
+using CoordinateTransformations
+using StaticArrays
+using Interpolations
+using OffsetArrays
+using FixedPointNumbers
+using Colors, ColorVectorSpace
+using Compat
+
+import Base: start, next, done, eltype, iteratorsize, size, length
 using Base: tail, Cartesian
 
-typealias FloatLike{T<:AbstractFloat} Union{T,Gray{T}}
-typealias FloatColorant{T<:AbstractFloat} Colorant{T}
-
 export
 
-    warp,
-    center,
     restrict,
-    imresize
+    imresize,
+    warp,
+    center
 
+include("autorange.jl")
```

Comment:
I had the same internal struggle. I even named it `corneriterator.jl` at first.  I then went with `autorange.jl` since it is the only function using `CornerIterator`, while other functions use `autorange`. A bit arbitrary, but at least it is some kind of reason :)